src/walletdb.cpp

```diff
@@ -1140,6 +1151,9 @@ std::list<CZerocoinMint> CWalletDB::ListMintedCoins(bool fUnusedOnly, bool fMatu
         CZerocoinMint mint;
         ssValue >> mint;
 
+        if(pwalletMain->IsCrypted() && !mint.Decrypt())
```

Comment:
Why do we want to decrypt the coins everytime we get the list of mints?

Reply:
does pwalletMain->IsCrypted() returning false not terminate the statement?